features.json

```diff
@@ -102,16 +107,18 @@
 		"Wasmtime": {
 			"url": "https://wasmtime.dev/",
 			"logo": "/images/bca.png",
-			"version": "0.20",
+			"version": "0.22",
 			"features": {
 				"bigInt": null,
 				"bulkMemory": true,
+				"moduleLinking": "--enable-module-linking",
 				"multiValue": true,
 				"mutableGlobals": true,
 				"referenceTypes": true,
 				"saturatedFloatToInt": true,
 				"signExtensions": true,
-				"simd": "--enable-simd"
+				"simd": "--enable-simd",
+				"threads": "--enable-threads"
```

Comment:
@fitzgen I just remembered why I didn't enable this in the first place. Is the issue I reported at https://github.com/bytecodealliance/wasmtime/issues/1658 resolved?

Reply:
We don't have shared memories yet, but we have partial support for atomics instructions. I was under the impression that the flags represented ongoing, work-in-progress support, which is why I added it. Maybe I misunderstood, and the flags should only be added when it is "done" but not on by default yet?

Reply:
> and the flags should only be added when it is "done" but not on by default yet?

Yeah, basically that.

Reply:
Sorry for the confusion!